Lib/idlelib/editor.py

```diff
@@ -1013,7 +1013,10 @@ def saved_change_hook(self):
             # that doesn't match platform conventions.
             title = short + " - " + long + _py_version
         elif short:
-            title = short
+            if short == "IDLE Shell":
+                title = short + _py_version
+            else:
+                title = short
```

Comment:
This 'else' branch is for macOS editor windows.  Currently has no version (Shell does, of course), so I think we should add it (with parens).
```suggestion
                title = short + _py_version
```